ext/date/tests/DateTimeZone_compare_basic1.phpt

```diff
@@ -71,6 +71,7 @@ compare Europe/Amsterdam with Europe/Berlin
 = bool(false)
 > bool(false)
 
-Warning: main(): Trying to compare different kinds of DateTimeZone objects in %s on line %d
-bool(false)
-Trying to compare uninitialized DateTimeZone objects
+Fatal error: Uncaught DateException: Cannot compare two different kinds of DateTimeZone objects in %s
+Stack trace:
+#0 {main}
+  thrown in %s
```

Comment:
Wrap this call in a try/catch and ``echo $e::class, ': ', $e->getMessage(), PHP_EOL;``?